sources/src/AuthPlugin.ts

```diff
@@ -153,6 +153,10 @@ class KeycloakPlugin {
 
     public async refresh(minValidity: number): Promise<void> {
         try {
+            if (!this.keycloak?.refreshToken) {
```

Comment:
@khoanguyen102 when we need this fix, we should revert my old fix with check for refreshToken in getToken as this is done in the refresh method now.

Reply:
we dont need it twice :)

Reply:
Ok Roli, I will update and retest it